backend/poetry.toml

```diff
@@ -1,3 +1,45 @@
-[virtualenvs]
-in-project = true
-create = true
+[tool.poetry]
+name = "dataprod"
+version = "0.1.0"
+description = ""
+authors = ["Bhargav Singapuri <bhsingapuri@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Django = "^4.0.7"
+psycopg2-binary = "^2.9.3"
+python-dotenv = "^0.19.2"
+pre-commit = "^2.17.0"
+pytest = "^7.0.1"
+pytest-cov = "^3.0.0"
+pytest-django = "^4.5.2"
+tqdm = "^4.63.0"
+pandas = "^1.4.1"
+whitenoise = "^6.0.0"
+setuptools = "^65.3.0"
+python-dateutil = "^2.8.2"
+django-debug-toolbar = "^3.2.4"
+psycopg2 = "^2.9.3"
+Jinja2 = "^3.1.2"
+protobuf = "^3.20"
+pip = "^22.1.2"
+install = "^1.3.5"
+jsonschema = "^4.6.0"
+pydantic = "^1.10.2"
+django-extensions = "^3.2.1"
+django-ninja = "^0.19.1"
+praw = "^7.6.1"
+nltk = "^3.7"
+ta = "^0.10.2"
+pgcli = "^3.5.0"
+google-search-results = "^2.4.1"
+tweepy = "^4.12.1"
+requests = "^2.28.1"
+
+[tool.poetry.dev-dependencies]
+autopep8 = "^1.6.0"
+ipython = "^8.4.0"
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
```

Comment:
Incorrectly changed file. This file is the poetry settings. `pyproject.toml` will contain the project settings